.github/workflows/continuous-integration.yml

```diff
@@ -15,15 +15,14 @@ jobs:
 
     strategy:
       matrix:
-        os:
-          - [self-hosted, Linux, X64]
-          - [self-hosted, macOS, ARM64]
-        rust_channel:
-          - stable
         include:
+          - os: [self-hosted, Linux, X64]
+            name: linux
           - os: [self-hosted, macOS, ARM64]
+            name: macos
             continue-on-error: true
 
+    name: build-and-test (${{ matrix.name }}, stable)
```

Comment:
As we've removed `rust_channel` from the parameters, shouldn't we even remove the `stable` at the end, if that's the only thing we test anyway ? (also because I suspect we would tweak the flake directly if we want to test other Rust channels, and not the workflow)

Reply:
Good point. Removed it from everywhere, including windows.